docs/en/Running-Alluxio-On-Docker.md

```diff
@@ -209,5 +161,45 @@ $ docker run -d --net=host --shm-size=1G \
              alluxio worker
 ```
 
-To prevent clients from attempting and failing short-circuit reads, the client hostname must
+To prevent clients from attempting and failing short-circuit reads, the client's hostname must
 be set to a value different from the worker's hostname. On the clients, configure `alluxio.user.hostname=dummy`.
+
+# Short-circuit operations
+
+In the tutorial, we set up a shared ramdisk between the host system and the worker container.
+This enables clients to perform read and write operations directly against the ramdisk instead
+of having to go through the worker process. A limitation of this approach is that it doesn't play
+well with setting worker container memory limits. Even though the ramdisk is mounted on the host,
+writes to it from the worker container will count towards the container's memory limit. To work
+around this limitation, you can share a domain socket instead of the ramdisk itself. The downside
+of using domain sockets is that they are computationally more expensive and suffer from lower
+write throughput.
+
+## Domain Socket
+
+From the host machine
```

Comment:
Maybe "From the host machine, create a directory for the shared domain socket."?

Reply:
done